AndelaOSP/art-dashboard
# [#157340065] User can add an asset type

## What does this PR do?
User can add asset type
## Description of Task to be completed?
- add component and container
- add async action to create asset type on the backend
- add reducer function to update frontend `store`
## How should this be manually tested?
`npm test` 
AND/OR
- Login
- Click on Assets on the Sidebar
- Click on the `+` icon next to the `Asset Types` column. A modal should pop up.
- Add the name of the asset type, select a sub category then click on save. The modal will close and a `success` message should appear on the top right corner of the page.
## What are the relevant pivotal tracker stories?
[157340065](https://www.pivotaltracker.com/story/show/157340065)
## Any background context you want to add?
n/a
## Important notes
n/a
## Packages installed
n/a
## Deployment note
n/a
## Related PRs branch | PR (branch_name) | (pr_link)
Branch | PR
------ | ------
other_pr_production | [link]()
other_pr_master | [link]()
## Todos
## Screenshots
<img width="1257" alt="screen shot 2018-07-04 at 18 11 29" src="https://user-images.githubusercontent.com/6042152/42284605-b8eb4812-7fb5-11e8-88e4-4fdd0b7af29e.png">
